### ISSUE TITLE
Flag images with no alt text at all

### ISSUE THREAD

**Issue by Mr0grog** [github/accessibility-alt-text-bot#23]
I really like the idea behind this action! However, the first few images I tried to test it with didn’t get flagged like I expected.

In the first instance, I happened to drag an image into the text area that had a filename that wasn’t “Screenshot <date>” (it was “edit-link.png”). I’m not sure if there’s a good way to identify stuff like this (maybe check alt-text with no spaces against a dictionary? tough to handle lots of languages, though…).

In the second instance, I tried just deleting the alt text entirely (all the variations of `<img alt="" ...>`, `<img ...>` (no alt attribute at all), and `![](<image url>)`). These seem more obviously straightforward to identify, and it would be nice if they were caught by this action.

---

As a side note, I thought I’d configured something wrong because of the above. The first thing I did was look at the action’s logging output, and it would have been a little bit helpful to have the two outputs (whether an image was flagged and the type of thing being checked) labeled so I knew what they were: https://github.com/github/accessibility-alt-text-bot/blob/b8484e124855b2e20bf7d01dd7fcdc2d536ab906/action.yml#L62-L63

(e.g. `echo "Detected bad alt text: ${flag}"` would have been helpful.)
